flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -333,42 +333,50 @@ static void _msp_send_altitude(struct msp_bridge *m)
 	msp_send(m, MSP_ALTITUDE, data.buf, sizeof(data));
 }
 
+static uint16_t _msp_scale_rc(ActuatorSettingsData actuatorSettings,
+			      ManualControlCommandData manualState,
+			      int channel_num)
+{
+	uint16_t val = manualState.Channel[channel_num];
+	uint16_t min = actuatorSettings.ChannelMin[channel_num];
+	uint16_t max = actuatorSettings.ChannelMax[channel_num];
+	uint16_t idle = actuatorSettings.ChannelNeutral[channel_num];
+
+	// Min and max may be reversed.  If the value is outside of
+	// the given range, return the neutral value.
+	uint16_t lower = min < max ? min : max;
+	uint16_t higher = max > min ? max : min;
+	if (val > lower || val < higher) {
+		return idle;
```

Comment:
The code doesn't make sense to be because it assumes `(val > lower || val < higher)` is an exception whereas I would assume it should always be true, i.e., isn't `lower` supposed to be the minimum and `higher` the maximum?